### DISCUSSION TITLE
Explorers

### DISCUSSION THREAD

**Discussion by AndreaMcP** [fedorovvl/tso_client#406] [Ideas]
Could we have a sort button so that the list order can be changed to the order the explorers will come back in? The multi-send is fantastic as I have over 200 explorers and sending them by hand is annoying.

**Comment by igorSof** [comment]
используйте наименование специалистов, переименуйте их в нужном порядке 
use the name of the specialists, rename them in the correct order

**Comment by fedorovvl** [comment]
what about sorting option in settings?
![изображение](https://github.com/user-attachments/assets/f8550997-6869-4295-9bb1-40c4273a8a03)
